### PR TITLE
Yamllint in verify step

### PR DESCRIPTION
**Pull request by obabec**

### CODE REVIEW

File: .github/config/yaml-config.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+config_data: |
```
Comment (kornys) [inline .github/config/yaml-config.yaml]: rename this file to something better. like `lint-config.yaml`

**Review by kornys — CHANGES_REQUESTED** [review]

File: .github/config/yaml-config.yaml (hunk 1 above)
Comment (obabec) [reply .github/config/yaml-config.yaml]: As i found out, default name of the file is .yamllint in root directory.

Comment (kornys) [reply .github/config/yaml-config.yaml]: I think this file should be moved into .github folder like I did for auto labeler

File: .github/workflows/yamllint.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  install:
```
Comment (Frawless) [inline .github/workflows/yamllint.yaml:12]:
```suggestion
  yaml-check:
```

File: argo/projects/tealc-ci.yaml [hunk 3]
```diff
@@ -16,7 +16,7 @@ spec:
       prune: true
       selfHeal: true
     syncOptions:
-    - CreateNamespace=true
+      - CreateNamespace=true
   destination:
     server: https://kubernetes.default.svc
-    namespace: tealc-ci
+    namespace: tealc-ciß
```
Comment (kornys) [inline argo/projects/tealc-ci.yaml:22]: WTF? :D

**Review by kornys — APPROVED** [review]

**Pull request merged**